ibm_catalog.json

```diff
@@ -387,7 +390,7 @@
             }
           ],
           "architecture": {
-            "descriptions": "This architecture supports deployment of Log Analysis and Cloud Monitoring instances on IBM Cloud and Activity Tracker event routing to a COS bucket target.",
+            "descriptions": "This architecture supports the deployment of Log Analysis and Cloud Monitoring instances on IBM Cloud. It also configures an Activity Tracker event routing to a COS bucket and optionally to a Log Analysis instance.",
```

Comment:
You can mention here -- It also configures the Activity Tracker event routing.

Then below  (L410)- add two points...one for COS bucket and one for Log Analysis.

Reply:
done